Frontend/timecamp/src/Components/Fields.jsx

```diff
@@ -1,419 +0,0 @@
-import React from "react";
-import styled from "styled-components";
-import {Link} from "react-router-dom"
-
-const FeatureSection = styled.div`
-  margin-bottom: 2rem;
-`;
-const Container = styled.div`
-  margin-top: 4rem;
-  margin-right: auto;
-  margin-left: auto;
-  padding-right: 15px;
-  padding-left: 15px;
-  width: 100%;
-  max-width: 1140px;
-`;
-
-const Row = styled.div`
-  margin: 0 auto;
-  display: flex;
-  align-items: center;
-  flex-wrap: wrap;
-  justify-content: space-between;
-`;
-
-const TextDiv = styled.div`
-  margin-bottom: 3rem;
-  text-align: center;
-  margin-top: 2rem;
-  position: relative;
-  width: 42%;
-  min-height: 1px;
-  padding-right: 15px;
-  padding-left: 15px;
-`;
-const ImgDiv = styled.div`
-  height: 650px;
-  display: flex;
-  align-items: center;
-  margin-bottom: 3rem;
-  text-align: center;
-  margin-top: 2rem;
-  position: relative;
-  width: 49%;
-  min-height: 1px;
-  padding-right: 15px;
-  padding-left: 15px;
-
-  & img {
-    width: 100%;
-    vertical-align: middle;
-    border-style: none;
-    text-align: center;
-  }
-`;
-const GP = styled.p`
-  font-size: 14px;
-  color: #25cf60;
-  font-weight: 700;
-  text-transform: uppercase;
-  margin-bottom: 2rem;
-  margin-top: 0;
-  text-align: left;
-`;
-const H2 = styled.h2`
-  padding-right: 2rem !important;
```

Comment:
using important is bad practice
https://stackoverflow.com/questions/3706819/what-are-the-implications-of-using-important-in-css